### DISCUSSION TITLE
Terre-Neuve

### DISCUSSION THREAD

**Discussion by sjinko** [CUPUM/ecometropole_laurentienne#52] [General]
Bonjour @CUPUM/ame6415_2022, je démarre cette discussion directement dans le dépôt pour échanger au sujet de notre voyage imminent à Terre-Neuve.

Déjà, j’aimerais savoir qui d’entrevous a un permis de conduire valide ?

**Comment by ArianeViens** [comment]
J'en ai un !

**Reply by Sarahprairie** [reply]
J'en ai un aussi

**Reply by AlexandraDesroches** [reply]
Moi aussi

**Reply by FlaviaCozac** [reply]
same

**Comment by AhmedBK95** [comment]
Moi aussi !

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Shin Alexandre Koseki ***@***.***>
Sent: Wednesday, October 12, 2022 5:30:00 PM
To: CUPUM/ecometropole_laurentienne ***@***.***>
Cc: Ahmed Kouider Benfodda ***@***.***>; Team mention ***@***.***>
Subject: [CUPUM/ecometropole_laurentienne] Terre-Neuve (Discussion #52)


Bonjour @CUPUM/ame6415_2022<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Forgs%2FCUPUM%2Fteams%2Fame6415_2022&data=05%7C01%7Cahmed.kouider.benfodda%40umontreal.ca%7C9d8ed18e6a9a436d7c4708daac98eb1d%7Cd27eefec2a474be7981e0f8977fa31d8%7C1%7C0%7C638012070022244367%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jKmPAj8pNktPpqKEJ1kRo3aI%2BSYKgUS8W5zGs5xbhy4%3D&reserved=0>, je démarre cette discussion directement dans le dépôt pour échanger au sujet de notre voyage imminent à Terre-Neuve.

Déjà, j’aimerais savoir qui d’entrevous a un permis de conduire valide ?

**Comment by MaryanneMonette** [comment]
Moi aussi!

Téléchargez Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : FlaviaCozac ***@***.***>
Envoyé : Wednesday, October 12, 2022 9:20:43 PM
À : CUPUM/ecometropole_laurentienne ***@***.***>
Cc : Maryanne Monette ***@***.***>; Team mention ***@***.***>
Objet : Re: [CUPUM/ecometropole_laurentienne] Terre-Neuve (Discussion #52)


same

**Comment by sjinko** [comment]
Ok, merci. On aura probablement besoin de deux personnes pour être co-pilotes et conduire les voitures occasionnellement. Pour des raisons d’assurances, ces personnes doivent avoir 25 ans ou plus. 
Les personnes intéressées peuvent nous écrire directement à shin.koseki@umontreal.ca et emile.maheu.forest@umontreal.ca

**Comment by Sarahprairie** [comment]
Bonjour, est-ce possible de déposer le plan du voyage/des activités à Terre-Neuve ?

**Reply by EmileForest** [reply]
@Sarahprairie c'est fait ! Avec quelques considéations pour la température et les équipements (cc @ArianeViens).

**Comment by ArianeViens** [comment]
J'ai une petite question : est-ce que les serviettes et débarbouillettes sont fournies à l'hébergement ?

**Reply by EmileForest** [reply]
@ArianeViens Ce n'est pas précisé, de mon côté je vais apporté [une serviette de ce type](https://www.sail.ca/en/sail-creek-microfibre-towel-medium-982189?utm_source=google&utm_medium=organic&utm_campaign=surfaces_across_google&utm_content=Outdoor%20Accessories%20(120BK)&gclid=CjwKCAjw-rOaBhA9EiwAUkLV4p8WEBMPTVE36z4asCIMQSeBgHPMCrkSaQlda4-HcEF52PwRAnhVVBoCb-wQAvD_BwE).

**Comment by MaryanneMonette** [comment]
Nous sommes en train de prévoir les repas de la semaine. On aurait besoin de connaitre les allergies. 
@sjinko  @EmileForest  En avez-vous ? 
Aussi, est-ce possible d'avoir les adresses courriels ou les noms des 3 autres étudiantes pour communiquer avec elles ? 

Merci !

**Reply by EmileForest** [reply]
Aucune allergie. Je mande tout, tout, tout.

**Comment by sjinko** [comment]
Aucune allergie de mon côté
________________________________

**Comment by sjinko** [comment]
Voici les adresses courriel

![05017638-982A-448A-83CE-D2A414DC11F7](https://user-images.githubusercontent.com/8132989/197346650-cf96482c-6beb-402e-b032-ad0c4599294f.jpeg)

________________________________

**Comment by ArianeViens** [comment]
Allo! Est-ce qu'il y aurait moyen à la place de Whatsapp, de se faire une conversation Messenger ? Parce que de toute façon il faut un réseau internet pour utiliser l'app, et l'application ne fonctionne pas pour tout le monde (dans mon cas, je ne suis pas capable de faire confirmer mon numéro de téléphone, donc l'app est inutilisable...)
